### ISSUE TITLE
Do random node picking for path to objective

### ISSUE THREAD

**Issue by Zezombye** [Zezombye/overwatch-bots-ai#5]
So that the bots don't always take the same paths.

**Comment by Zomg15** [comment]
my idea: have a couple "path nodes" that each define a "route" to the point
when the bots go to the point, they will pick a random path node, go there, and then go to the point from there
for example, on nepal village, that could be the tiny room in the top path, or the large choke on the main path

**Comment by Zezombye** [comment]
Yup that was exactly what i was thinking, great minds think alike owo
